src/main/KeyboardControls.tsx

```diff
@@ -130,30 +119,22 @@ const KeyboardControls: React.FC = () => {
   })
 
   const render = () => {
-    if (keyMap && Object.keys(keyMap).length > 0) {
-
-      const obj: Record<string, Array<KeyMapDisplayOptions>> = {}
-      obj[t("keyboardControls.defaultGroupName")] = []    // For keys without a group
-
-      // Sort by group
-      for (const [, value] of Object.entries(keyMap)) {
-        if (value.group) {
-          if (obj[value.group]) {
-            obj[value.group].push(value)
-          } else {
-            obj[value.group] = [value]
-          }
-        } else {
-          obj[t("keyboardControls.defaultGroupName")].push(value)
-        }
-      }
+    if (KEYMAP && Object.keys(KEYMAP).length > 0) {
 
       const groups: JSX.Element[] = [];
-      for (const key in obj) {
-        if (obj[key].length > 0) {
-          groups.push(<Group name={key as ParseKeys} entries={obj[key]} key={key}/>);
-        }
-      }
+      Object.entries(KEYMAP).forEach(([key, value], index) => {
+        const entries : { [key: string]: string[][] } = {}
+        Object.entries(value).forEach(([, value]) => {
+          const sequences = value.key.split(",").map(item => item.trim())
+          const lol: string[][] = []
+          Object.entries(sequences).forEach(([, value]) => {
+            const keys = value.split("+").map(item => item.trim())
+            lol.push(keys)
+          })
+          entries[value.name] = lol
```

Comment:
`lol` should get a more descriptive name. Or, just rewrite it to now require a temporary variable.

```tsx
entries[value.name] = Object.entries(sequences).map(...);
```